client/src/component/SearchBar.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+export default function SearchBar() {
```

Comment:
```suggestion
export default function KeywordSearchBar() {
```

KeywordSearchBar 면 이해할 수 있을 것 같아요.